src/actionCable/middleware.js

```diff
@@ -0,0 +1,89 @@
+import Config from 'react-native-config';
+
+import {
+  createConsumer,
+  disconnectActionCable,
+  receiveMessage,
+  sendMessage,
+  subscribe,
+  unsubscribe,
+} from 'actions/chatActions';
+import { clearOnboarding } from 'actions/onboardingActions';
+import actionCable from './actionCable';
+import { CHAT_CHANNEL, SEND_MESSAGE } from 'constants/chat';
+import { applyQueryParams } from 'utils/helpers';
+
+const actionCableMiddleware = (() => {
+  let cable = null;
+  const chatChannels = [];
+
+  return store => next => action => {
+    switch (action.type) {
+      case createConsumer.toString(): {
+        cable && cable.disconnect();
```

Comment:
```suggestion
        cable?.disconnect();
```

Check for this on this file and other files too

Reply:
optional chaining FTW